app/permissions_migrations.go

```diff
@@ -516,6 +516,15 @@ func (a *App) getAddManageSharedChannelsPermissionsMigration() (permissionsMap,
 	}, nil
 }
 
+func (a *App) getBillingPermissionsMigration() (permissionsMap, error) {
+	return permissionsMap{
+		permissionTransformation{
+			On:  isRole(model.SYSTEM_ADMIN_ROLE_ID),
+			Add: []string{model.PERMISSION_SYSCONSOLE_READ_BILLING.Id, model.PERMISSION_SYSCONSOLE_WRITE_BILLING.Id},
+		},
+	}, nil
+}
```

Comment:
Does this need added to any of the new roles?

Reply:
I wasn't sure either but if it's not there by default , I was thinking it's not too hard for the system admin to assign it to those roles.

Reply:
@marianunez Any comments on this? Do you think by default any other role should have this permission?

Reply:
For the purposes of this PR, I agree that Admin would be enough and they can grant permissions to other roles as needed. We do have the concept Billing Admins as a feature in the roadmap so I will work with PM to see how this would fit with these new System Roles and if we would want to modify any others as well.

Reply:
Cool, thanks for the quick response Maria!